handlers/metrics.go

```diff
@@ -227,6 +233,8 @@ func (mh *metricHandler) MetricsBountiesCount(w http.ResponseWriter, r *http.Req
 		return
 	}
 
+	request.Providers = providers
```

Comment:
Don't pass the providers to the request variable, get it from the request.Query

Reply:
Addressed